includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -358,6 +361,11 @@ public function getExploreMenu(): array {
 				],
 			];
 
+			// if community has store data, add to explore dropdown
+			if ( $storeData ) {
```

Comment:
Can we make this check more robust? If the API changes do something that might not display correctly it will cause errors for lots of users. Let's be 100% certain that the API is returning something we can handle.

Reply:
`formatFandomStoreData` probably is the right place because we are already looking at the specific properties. Maybe a function called in there

Reply:
Added check.